core/pom.xml

```diff
@@ -167,6 +167,31 @@
 					</dependency>
 				</dependencies>
 			</plugin>
+			<plugin>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
```

Comment:
Isn't the version missing here?

Reply:
So 6.4.0 als long we are under jdk17

Okay?

Reply:
The build already runs with a jdk-17, so 7.0.0 should be fine.
It's just that the built artifacts target Java-11 but that shouldn't matter for the use of BND during the build.

We just really should the specify the version, because currently the build warns:
```
Warning:  Some problems were encountered while building the effective model for org.eclipse.dash:org.eclipse.dash.licenses.core:jar:1.1.1-SNAPSHOT
Warning:  'build.plugins.plugin.version' for biz.aQute.bnd:bnd-maven-plugin is missing. @ line 169, column 12
Warning:  
Warning:  It is highly recommended to fix these problems because they threaten the stability of your build.
Warning:  
Warning:  For this reason, future Maven versions might no longer support building such malformed projects.
```

Reply:
updated the version